pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc

```diff
@@ -0,0 +1,19 @@
+_module-type: PROCEDURE
+[id="proc_submitting-documentation-feedback_{context}"]
+= Submitting documentation feedback
+
+[role="_abstract"]
+If you encounter any documentation bugs or have any requests for additions or changes to the {ProductName} end-user documentation, you can open a Jira ticket to create a request for the documentation team.
+
+Do not submit documentation feedback to submit a bug or functionality request for Pantheon.
+
+.Procedure
+
+. Log in to link:https://projects.engineering.redhat.com/projects/FCCEUD/issues[the Jira project for the Pantheon 2 end-user documentation (FCCEUD)]. This is the internal Red Hat instance of Jira, so you need to be connected to the Red Hat network in an office or using the VPN.
+. Click *Create*.
+. Fill out the form as follows:
++
+Project:: *FCC - Pantheon V2 End-User Documentation* (FCCEUD)
+Issue Type:: *Bug* or *Task*
+Summary:: This is the title of the ticket. Enter *`[FEEDBACK]`* Followed by a short description of the issue. For example: `[FEEDBACK] Broken link in module "Module types"`
+Description:: Enter information that fully explains the feedback. Include as much detail as you can, including any questions you would like answered.
```

Comment:
I'd add the following line:
`. Click *Create*.`

Reply:
Done